odl/solvers/iterative/statistical.py

```diff
@@ -36,9 +36,9 @@ def mlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
 
     Attempts to solve::
 
-        max_x L(data | x)
+        max_x L(x | data)
 
-    where ``L(data, | x)`` is the likelihood of ``data`` given ``x``. The
+    where ``L(x | data)`` is the likelihood of ``data`` given ``x``. The
```

Comment:
Expression is correct, but the following explanation is still the old one. Should be "the likelihood of ``x`` given ``data``.